R/pkg/R/DataFrame.R

```diff
@@ -2973,15 +2974,51 @@ setMethod("describe",
             dataFrame(sdf)
           })
 
+#' summary
+#'
+#' Computes specified statistics for numeric and string columns.
+#'
+#' Available statistics are:
+#'
+#' - count
+#' - mean
+#' - stddev
+#' - min
+#' - max
```

Comment:
these bullets and whitespaces get collapsed by roxgyen2 - try itemize/item https://cran.r-project.org/web/packages/roxygen2/vignettes/rd.html